### PR TITLE
feat(vehicle_info_util): add max_steer_angle

### PR DESCRIPTION
**Pull request by takayuki5168**
## Description

Max steer angle is added in the `vehicle_info_util` package.
Each vehicle description repositories has to be changed as well.

## Related links

Resolves https://github.com/autowarefoundation/autoware.universe/issues/535

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].
- [ ] The PR has been properly tested.
- [ ] The PR has been reviewed by the code owners.

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.
- [ ] The PR is ready for merge.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: vehicle/vehicle_info_util/config/vehicle_info.param.yaml [hunk 1]
```diff
@@ -9,3 +9,4 @@
     left_overhang: 0.1 # between left wheel center and vehicle left
     right_overhang: 0.1 # between right wheel center and vehicle right
     vehicle_height: 2.5
+    max_steer_angle: 0.70 # [rad]
```
Comment (tkimura4) [inline vehicle/vehicle_info_util/config/vehicle_info.param.yaml:12]: Could you apply pre-commit ?

**Comment by kenji-miyake** [comment]
pre-commit.ci run

File: vehicle/vehicle_info_util/config/vehicle_info.param.yaml (hunk 1 above)
Comment (kenji-miyake) [reply vehicle/vehicle_info_util/config/vehicle_info.param.yaml:12]: I'm wondering why it wasn't autofixed. :thinking:

Comment (tkimura4) [reply vehicle/vehicle_info_util/config/vehicle_info.param.yaml]: [e7b3df0](https://github.com/autowarefoundation/autoware.universe/pull/740/commits/e7b3df03471a6265b7aa99b74d70b3ab60498835)

Comment (takayuki5168) [reply vehicle/vehicle_info_util/config/vehicle_info.param.yaml]: I did, manually.

**Comment by takayuki5168** [comment]
@TakaHoribe @tkimura4 @kenji-miyake 
I agreed with the following Horibe-san's comment. Therefore this PR is now ready for review.
https://github.com/autowarefoundation/autoware.universe/issues/535#issuecomment-1109984238_

**Comment by TakaHoribe** [comment]
This PR looks good to me. Please merge the associated modification for descriptions in the same time.

**Comment by kenji-miyake** [comment]
@takayuki5168 I respect Taka's opinion. :+1:

**Comment by yukkysaito** [comment]
It is good.
LGTM

**Pull request merged**